common/stringify.js

```
@@ -0,0 +1,74 @@
+
+  switch (typeof obj) {
+    case 'string':
+      return "'" + obj + "'"
+    case 'undefined':
+      return 'undefined'
+    case 'function':
+      return obj.toString().replace(/\{[\s\S]*\}/, '{ ... }')
+    case 'boolean':
+      return obj ? 'true' : 'false'
+    case 'object':
+      var strs = []
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)